src/lib/data.nix

```diff
@@ -32,11 +152,11 @@ let
   commands = {
     asciidoc = "asciidoctor -b xhtml5 -s -a showtitle -o-";
     markdown = "multimarkdown";
-    org-mode = "bash ${pkgs.styx}/share/styx/tools/emacs-compile";
+    org-mode = "${org-compile} compile-org-file";
```

Comment:
I need a separate command since the order of the given params is important, so I can't just add --file ${fileData.path}  at the end of the command. The last thing must be -f "function-to-call".

Reply:
Ok, this is just a detail that can be cleaned up later. Thanks for the explanation.